sdk/ai/ai-projects/samples-dev/agents/file_search.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { AIProjectsClient, isOutputOfType, MessageContentOutput, MessageImageFileContentOutput, MessageTextContentOutput } from "@azure/ai-projects";
+import { delay } from "@azure/core-util";
+import { DefaultAzureCredential } from "@azure/identity";
+
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+dotenv.config();
+
+const connectionString = process.env["AZURE_AI_PROJECTS_CONNECTION_STRING"] || "<endpoint>>;<subscription>;<resource group>;<project>";
+
+export async function main(): Promise<void> {
+  const client = AIProjectsClient.fromConnectionString(connectionString || "", new DefaultAzureCredential());
+
+  // Upload file
+  const localFileStream = fs.createReadStream("local_file.txt");
+  const file = await client.agents.uploadFile(localFileStream, "assistants", "sample_file_for_upload.txt");
+  console.log(`Uploaded file, ID: ${file.id}`);
+
+  // Create vector store
+  const vectorStore = await client.agents.createVectorStore({ file_ids: [file.id], name: "my_vector_store" });
```

Comment:
Does the vector store create immediately and available for immediate consumption?

Reply:
yes it is immediately available